test_data/std/binary_test.asm

```diff
@@ -1,7 +1,7 @@
 use std::machines::binary::Binary;
 
 
-machine Main with degree: 262144 {
+machine Main with degree: 196608 {
```

Comment:
I'm not sure this makes a difference?

Reply:
ah wait, does this even work if it's not a power of two?

Reply:
looks like it does, even in the backends?

Reply:
It gets padded to the next power of 2

Reply:
Then it's a nice test...